src/components/pages/packages/flyouts/newPackage/newPackage.container.js

```diff
@@ -3,11 +3,21 @@
 import { connect } from 'react-redux';
 import { translate } from 'react-i18next';
 import { NewPackage } from './newPackage';
-import { epics as packagesEpics } from 'store/reducers/packagesReducer';
+import {
+  getCreatePackageError,
+  getCreatePackagePendingStatus,
+  epics as packagesEpics
+} from 'store/reducers/packagesReducer';
+
+// Pass the global info needed
+const mapStateToProps = state => ({
+  isPending: getCreatePackagePendingStatus(state),
+  error: getCreatePackageError(state)
+});
 
 // Wrap the dispatch methods
 const mapDispatchToProps = dispatch => ({
-  createPackage: (packageObj) => dispatch(packagesEpics.actions.createPackage(packageObj))
+  createPackage: packageModel => dispatch(packagesEpics.actions.createPackage(packageModel))
 });
 
-export const NewPackageContainer = translate()(connect(null, mapDispatchToProps)(NewPackage));
+export const NewPackageContainer = translate()(connect(mapStateToProps, mapDispatchToProps)(NewPackage));
```

Comment:
The flyouts for adding new devices and rules are named widgetNew (instead of newWidget) as here. Please be consistent.

Reply:
widgetNew is confusing. We should changer the others.

Reply:
Be my guest. However, note that all the flyouts are named that way... so its not a matter of changing just a couple. My point is to be consistent. Don't deviate because you think the others are wrong without also changing them. Just my 2 cents.